crates/sui-protocol-config/src/lib.rs

```diff
@@ -113,6 +113,7 @@ pub struct Error(pub String);
 struct FeatureFlags {
     // Add feature flags here, e.g.:
     // new_protocol_feature: bool,
+    package_upgrades_allowed: bool,
```

Comment:
tiny nit: Do we need the `_allowed`? `package_upgrades` is the feature.
```suggestion
    package_upgrades: bool,
```

Reply:
I don't think we do. I'll rename it :)